src/services/dashboardService/SystemUsageSummary.ts

```diff
@@ -0,0 +1,82 @@
+import { AxiosInstance, AxiosStatic } from 'axios';
+import { RefObject } from 'react';
+import { AppSession } from './../app-session';
+import { ResultWithoutExpectedErrors } from '../../doppler-types';
+
+type DefaultType = boolean | null | undefined;
+
+export interface SystemUsageSummary {
+  hasListsCreated: DefaultType;
+  hasDomainsReady: DefaultType;
+  hasCampaingsCreated: DefaultType;
+  hasCampaingsSent: DefaultType;
```

Comment:
Creo que con esto es suficiente, puede ser?

```suggestion
  hasListsCreated: boolean?;
  hasDomainsReady: boolean?;
  hasCampaingsCreated: boolean?;
  hasCampaingsSent: boolean?;
```